public/index.html

```diff
@@ -4,10 +4,11 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+    <link rel="stylesheet" type="text/css" href="https://cloud.typography.com/7761236/7617792/css/fonts.css" />
     <script src="https://apis.google.com/js/platform.js"></script>
     <style>
       * {
-        font-family: Arial, Helvetica, sans-serif;
+        font-family: "Gotham SSm A", "Gotham SSm B", sans-serif;
```

Comment:
Sure you don't want a fallback?

Reply:
TBH, sans-serif is probably fine, whatever we'd define would likely be the default for that (Arial, Helvetica)

Reply:
(But I'll ask Lou when we're back in the office)